README.md

```diff
@@ -300,6 +300,7 @@ kover {
     intellijEngineVersion.set("1.0.637")    // change version of IntelliJ agent and reporter
     jacocoEngineVersion.set("0.8.7")        // change version of JaCoCo agent and reporter
     generateReportOnCheck.set(true)         // false to do not execute `koverReport` task before `check` task
+    disabledModules = setOf()               // setOf("module-name") to disable coverage for module with name `module-name`
```

Comment:
In this case, it is necessary to carry out a global renaming in the entire project.
It is worth doing this when all opened PRs will be merged.

Reply:
You can leave internal functions (like 'Project.applyToModule') as-is for now, but public-facing configuration properties should not confuse users.

Reply:
It is necessary to change the documentation so that the terminology from the README does not contradict the terminology from the DSL plugin.
You will also need to carefully check all occurrences of the word `module`.

Reply:
OK, let's make it a separate PR